public/app/features/playlist/PlaylistPage.tsx

```diff
@@ -22,10 +33,12 @@ interface Props extends ConnectedProps, GrafanaRouteComponentProps {}
 export const PlaylistPage: FC<Props> = ({ navModel }) => {
   const [searchQuery, setSearchQuery] = useState('');
   const [startPlaylist, setStartPlaylist] = useState<PlaylistDTO | undefined>();
+  const [playlistToDelete, setPlaylistToDelete] = useState<PlaylistDTO | undefined>();
+  const [forcePlaylistsFetch, setforcePlaylistsFetch] = useState(0);
 
   const { value: playlists, loading } = useAsync(async () => {
     return getBackendSrv().get('/api/playlists', { query: searchQuery }) as Promise<PlaylistDTO[]>;
-  });
+  }, [forcePlaylistsFetch]);
```

Comment:
Yes - you download the list of playlists, and after that you only update local state of the component.

Reply:
Couldn't another user have deleted playlists and we want to reflect that too?

Reply:
Probably tru @hugohaggmark. Question how often the playlists are deleted and whether or not this will be an issue. Dunno to be honest. My gut feeling is that this ain't the mostly used list in Grafana.

Reply:
I agree with you Domink, but I think we should avoid having a disconnected frontend state.

Reply:
That's totally fine with me @hugohaggmark . Let's keep it as it is implemented atm - since this endpoint ain't used too much probably, re-fetching list should still be fine.

Reply:
@thisisobate can we expose the `getBackendSrv().get(` from the api file public/app/features/playlist/api.ts instead? Much like you did with the delete function?